instance/static/html/instance/index.html

```diff
@@ -10,7 +10,7 @@
   <!-- Instances List -->
   <div class="large-3 columns">
     <ul class="side-nav side-instance-list">
-      <li ng-repeat="instance in instanceList | filter: {is_shut_down: false}"
+      <li ng-repeat="instance in instanceList | filter: {is_archived: false}"
```

Comment:
Thanks for this change.  I think it will make a significantly speed things up.  The serialised data for our production instance is more than 1K per instance, so including archived instances results in a 300K response everytime the list is updated.  The database queries will also be significantly faster if we first filter out everything we don't need.